docs/src/main/asciidoc/_cloud/Getting_Started.adoc

```diff
@@ -0,0 +1,147 @@
+# WildFly on OpenShift - Developer Usage Guide
+
+This guide is designed to assist you through obtaining an OpenShift instance and installing WildFly, complete with a deployed application. It is not intended to cover all possible configurations, as links to more detailed documentation will be provided.
+
+### Obtaining an OpenShift Instance
+There are many options available for running OpenShift. This guide will mention two.
+
+##### Code-ready-containers
+If you wish to run a local development environment, then https://developers.redhat.com/products/openshift-local/overview[CRC] is likely to be a good starting point.
+
+##### Developer sandbox
+https://www.redhat.com/en/technologies/cloud-computing/openshift/try-it[The Red Hat OpenShift Developer Sandbox is a hosted environment], and is free to use for development purposes.
+
+This guide will focus on usage of the hosted developer sandbox environment. The sandbox should be provisioned from the page above before proceeding further.
+
+### OpenShift
+In order to use OpenShift effectively, in addition to a running instance, you will also need the OpenShift client tools (`oc` command). This may be obtained from your package manager or downloaded from your OpenShift sandbox (in your control panel, clicking on the question mark icon at the top right of the console presents a drop-down menu where `Command line tools` takes you to the latest version of `oc`).
+
+Once you have downloaded and installed/unpacked the client, you can authenticate to your running sandbox OpenShift instance by using the `Copy Login Command` from the upper right menu in the sandbox user-interface (your username is displayed there.)
+
+Example:
+```
+$ oc login --token=sha256~b-3av_0Lp8wmJu0G3WXzdfds34454mkvWjRRESd_jHTA --server=https://api.sandbox.x8i5.p1.openshiftapps.com:6443
+```
+
+We'll be using the `someuser-dev` project for this example:
```

Comment:
I think it would be good to indicate that the user should replace "someuser" with their own username.